src/lib.rs

```diff
@@ -151,46 +216,47 @@ impl Collector {
     /// # }
     /// ```
     pub fn describe(&self) {
-        let prefix = &self.prefix;
+        let metrics = self.metrics.as_ref();
+
         describe_gauge!(
-            format!("{}process_cpu_seconds_total", prefix),
+            metrics.cpu_seconds_total.clone(),
```

Comment:
This is just a suggestion, but could you consider rephrasing it as `Arc::clone(&metrics.cpu_seconds_total)` instead? I lean towards this approach because the `.clone()` operation is usually resource-intensive, and I want to easily distinguish `Arc::clone` from other uses of `.clone()`.